cypress/component/ColumnAnnotationCard.cy.tsx

```diff
@@ -0,0 +1,134 @@
+      'have.class',
+      'Mui-selected'
+    );
+    cy.get('[data-cy="1-column-annotation-card-data-type-categorical-button"]').should(
+      'not.have.class',
+      'Mui-selected'
+    );
+  });
+  it('Fires the onDescriptionChange event handler with the appropriate payload when the save button is clicked', () => {
+    const spy = cy.spy().as('spy');
+    cy.mount(
+      <ColumnAnnotationCard
+        id={props.id}
+        header={props.header}
+        description={props.description}
+        dataType={props.dataType}
+        standardizedVariable={props.standardizedVariable}
+        standardizedVariableOptions={props.standardizedVariableOptions}
+        onDescriptionChange={spy}
+        onDataTypeChange={props.onDataTypeChange}
+        onStandardizedVariableChange={props.onStandardizedVariableChange}
+      />
+    );
+    cy.get('[data-cy="1-column-annotation-card-edit-description-button"]').click();
+    cy.get('[data-cy="1-column-annotation-card-description-input"]').clear();
+    cy.get('[data-cy="1-column-annotation-card-description-input"]').type('new description');
+    cy.get('[data-cy="1-column-annotation-card-save-description-button"]').click();
+    cy.get('@spy').should('have.been.calledWith', 1, 'new description');
+  });
+  it('Fires the onDataTypeChange event handler with the appropriate payload when the data type is toggled', () => {
+    const spy = cy.spy().as('spy');
+    cy.mount(
+      <ColumnAnnotationCard
+        id={props.id}
+        header={props.header}
+        description={props.description}
+        dataType={props.dataType}
+        standardizedVariable={props.standardizedVariable}
+        standardizedVariableOptions={props.standardizedVariableOptions}
+        onDescriptionChange={props.onDescriptionChange}
+        onDataTypeChange={spy}
+        onStandardizedVariableChange={props.onStandardizedVariableChange}
+      />
+    );
+    cy.get('[data-cy="1-column-annotation-card-data-type-continuous-button"]').click();
+    cy.get('@spy').should('have.been.calledWith', 1, 'Continuous');
+  });
+  it('Fires the onStandardizedVariableChange event handler with the appropriate payload when the standardized variable is changed', () => {
+    const spy = cy.spy().as('spy');
+    cy.mount(
+      <ColumnAnnotationCard
+        id={props.id}
+        header={props.header}
+        description={props.description}
+        dataType={props.dataType}
+        standardizedVariable={props.standardizedVariable}
+        standardizedVariableOptions={props.standardizedVariableOptions}
+        onDescriptionChange={props.onDescriptionChange}
+        onDataTypeChange={props.onDataTypeChange}
+        onStandardizedVariableChange={spy}
+      />
+    );
+    cy.get('[data-cy="1-column-annotation-card-standardized-variable-dropdown"]').type(
+      'age{downarrow}{enter}'
+    );
+    cy.get('@spy').should('have.been.calledWith', 1, { identifier: 'nb:Age' });
+  });
```

Comment:
Let's add test cases also for "if I set the StandardizedVariable to XYZ in the store, it'll show up with the label" - that would have caught the current bug

Reply:
If not feasible for the AnnotationCard because it doesn't use the store directly, then maybe as a test of the parent component where we also load the AnnotationCard child component to ensure that setting the (mocked) store leads to the expected UI rendering outcome of "I now see the label of the Variable selected, not the identifier"